sbin/pfdhcplistener

```diff
@@ -107,6 +109,8 @@ sub reload_config {
     }
     elsif ( pf::cluster::is_vip_running($interface) ) { 
         $process = $TRUE;
+        $is_clustered = $TRUE;
+        $holds_vip = $TRUE;
```

Comment:
You're never setting this variable back to false so it seems that if you own the VIP once, this flag will stay true even if you lose the VIP